docs/PacketFence_Aerohive_Quick_Install_Guide.asciidoc

```diff
@@ -18,433 +18,267 @@ include::includes/global-attributes.asciidoc[]
 
-You'll be able to login to the PacketFence web administration interface with the credentials created in Step 4.
+* Deploye the _Optionnal Setting(not supported by RADIUS Proxy)_ section and * [x] check the _Permit Dynamic Change of Authorization Message (RFC 3576)_
+* Click on *Save*
+:
+* Next to your _SSID Name_ Click on the *<RADIUS Setting>*, Click *OK*
 
-Services status will help you monitor if everything goes as expected. If not, you'll see which service is in trouble and the log output will help you determine the problem that occurs.
+'''
 
-Configuring Aerohive access point in PacketFence
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Now that you have a fully functional PacketFence installation, you'll need to add the Aerohive AP to the PacketFence switches database for correct integration.
-
-To do so, login to the PacketFence web administration interface if it is not already done. Click on the 'Configuration' tab and select the Switches section.
-
-We'll use the clone function to add a new switch (the AeroHive AP) to the PacketFence database. With the mouse pointer, go over the default switch and you'll see a set of icons appearing at the left of it. Click on the second one (a paper sheet with a plus sign).
-
-Configure the switch like the following:
-
-image::docs/images/aerohive-pf-config-1.png[scaledwidth="100%",alt="General switch settings"]
-
-image::docs/images/aerohive-pf-config-vlan.png[scaledwidth="100%",alt="VLAN switch settings"]
-
-image::docs/images/aerohive-pf-config-snmp.png[scaledwidth="100%",alt="SNMP switch settings"]
-
-Modify your RADIUS secret.
-
-Click 'Add switch'
-
-The newly added access point should show up in the list.
-
-PacketFence configuration is done. You may now reboot the demonstration PC (leave the USB key in). Once rebooted, the web browser should open in the PacketFence web administration interface.
-
-<<<
-
-WebAuth Enforcement
-^^^^^^^^^^^^^^^^^^^
-
-Step 2: Networks
-++++++++++++++++
-This step will ask you to statically configure your network interfaces (note that DHCP interfaces configuration is not supported yet).
-
-Depending on the choice(s) made on step 1, you'll have to configure the required types of interface. The web interface will list all currently installed network interfaces on the system. An IP and a netmask will be visible if the network interface is configured (either by DHCP or already manually configured). You can edit those ones, create/delete VLANs on physical interfaces and enable/disable an interface. Note that theses changes are effective on the moment you make them. Persistence will be written only for ENABLED interfaces.
-
-In all time, you'll need to set a Management interface.
-
-Required interface types for WebAuth enforcement:
-
-    portal
 
-Note that you can only set ONE (1) management interface. This one will work for both in the case you choose both modes.
+We will create the default VLAN to be assign by the AP, when a new Terminal get in the SSID ::
```

Comment:
to be assigned
a new Terminal? I guess you mean "when a new endpoint connects to the SSID:"